model/qti/parser/IncludedElementIdsExtractor.php

```diff
@@ -26,44 +26,32 @@
 use oat\tao\model\media\TaoMediaResolver;
 use oat\taoQtiItem\model\qti\Item;
 use oat\taoQtiItem\model\qti\XInclude;
-use tao_helpers_Uri;
 
+/**
+ * @deprecated Use \oat\taoQtiItem\model\qti\parser\ElementIdsExtractor
```

Comment:
we should delete it

Reply:
You mean the class? But then we would have to bump a major version just because of this.

Reply:
yes, but this class has been added last sprint and it is not used anywhere else 🤞 . If we keep it, it is going to create legacy dead code. 
It has been merged too fast...

Reply:
I know, but are we able to take that risk?

Reply:
yes, because after release, a regression test is performed before going to prod. We can safely accept it. However, we can merge that at begin of a sprint to have time to discover eventual issue. It is reducing the risk

Reply:
Ok. I removed